tox.ini

```diff
@@ -9,7 +9,7 @@ commands =
 skip_install = true
 deps =
     coverage
-    setuptools==33.1.1
+    setuptools==39.0.1
```

Comment:
Drive-by fix: this needs to match versions.cfg

Reply:
Good catch.